wolfcrypt/src/evp.c

```diff
@@ -6803,6 +6803,833 @@ void wolfSSL_EVP_PKEY_free(WOLFSSL_EVP_PKEY* key)
         }
     }
 }
+#if defined(OPENSSL_EXTRA)
+/* Converts byte data into 2 digit hex code
+ * then put them to the specified buffer.
+ */
+static int ToHex( byte in, byte* hex )
+{
+    byte UpNibble,LwNibble;
+    byte HexTbl[16] = { '0','1','2','3','4','5','6','7',
+                        '8','9','a','b','c','d','e','f' };
+
+    if ( hex == NULL )
+        return 0;
+
+    UpNibble = (in >> 4) & 0x0f;
+    LwNibble =  in & 0x0f;
+    *hex++ = HexTbl[UpNibble];
+    *hex   = HexTbl[LwNibble];
+    return 2;
+}
+/* Converts input value "i" to upto five digit decimal
+ * and copys to the specified buffer.
+ */
+static int ToDec(word32 in, byte* hex)
+{
```

Comment:
Same above.

Reply:
ToDec provides conversion to decimal strings, but Base16_Decode does not. So it seems that ToDec cannot be replaced by Base16_Decode.

Reply:
if Base16_Decode will not work here then can we use some of the mp functions? Such as mp_todecimal? Am trying to avoid adding in a new number format conversion function.

Reply:
Thanks. mp_todecimal works! Removed ToDec.